Completion/Zsh/Context/_subscript

```diff
@@ -98,6 +98,9 @@ elif [[ ${(Pt)${compstate[parameter]}} = array* ]]; then
   while _tags; do
     if _requested indexes; then
       ind=( {1..${#${(P)${compstate[parameter]}}}} )
```

Comment:
When the array paramater is empty, this results in:

```shell
ind=( {1..${#${(P)${compstate[parameter]}}}} )
ind=( {1..0} )
ind=( 1 0 )
```

Which is not appropriate for the completion context.